_posts/2022-08-08-july-rubygems-updates.md

```diff
@@ -0,0 +1,41 @@
+---
+title: July 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Central](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in July.
+
+# RubyGems News
+
+This month in RubyGems, we released new versions of RubyGems [3.3.18](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3318--2022-07-14), [3.3.19](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3319--2022-07-27) and Bundler [2.3.18](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2318-july-14-2022), [2.3.19](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2319-july-27-2022).
+
+The following is a non-exhaustive list of other improvements included in the above releases (see the changelog for further information):
+ 
+- fixed an issue with Bundler on Windows that allows the new `x64-mingw-ucrt`, the default on **Ruby 3.1** to work seamlessly with the existing `platforms DSL` - [#5655](https://github.com/rubygems/rubygems/pull/5655). 
+- improved the  `bundler/setup` speed by lazily checking for missing platform-specific variants if nothing else has changed - [#5695](https://github.com/rubygems/rubygems/pull/5695).
+- fixed some issues on `truffleruby` caused by other bug - [#5694](https://github.com/rubygems/rubygems/pull/5694), [#5746](https://github.com/rubygems/rubygems/pull/5746)
+- fixed confusing permission error when copying compact index cache  - [#5709](https://github.com/rubygems/rubygems/pull/5709).
+- fixed an issue with Bundler printing the bug report template when it should give a better error and not suggest a bug - [#5726](https://github.com/rubygems/rubygems/pull/5726).
+- improved `gem not found` error messages to include expected source - [#5729](https://github.com/rubygems/rubygems/pull/5729).
+- merged a PR to fix `gem update --system` errors in some edge cases - [#5728](https://github.com/rubygems/rubygems/pull/5728), [#5737](https://github.com/rubygems/rubygems/pull/5737).
+- made RubyGems requiring `fileutils` lazier to avoid some flaky test failures - [#5738](https://github.com/rubygems/rubygems/pull/5738).
```

Comment:
I would probably spare the above entry (since it's unclear whether it was a real end user issue), and instead include links to the following three features (better at the top, before other entries)

```
- display mfa warnings on `gem signin`, to support work done on the rubygems.org side - [#5590](https://github.com/rubygems/rubygems/pull/5590).
- add the long requested feature of allowing to install gems from specific groups only - [#5579](https://github.com/rubygems/rubygems/pull/5759)
- extend `gem` DSL with a `force_ruby_platform` option - [#4049](https://github.com/rubygems/rubygems/pull/4049)
```

Reply:
Thank you!! will add these entries to the post.